e2e/npm_translate_lock_patch/portal/.bazelignore

```diff
@@ -0,0 +1 @@
+portal/node_modules
```

Comment:
Is there something meaningful about the term "portal"? If not, maybe we can just call it "subdir"?